Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -212,17 +210,22 @@ class PDHG(Algorithm):
 
     Note
     ----
-
     The case where both functions are strongly convex is not available at the moment.
 
+    """
 
-    .. todo:: Implement acceleration of PDHG when both functions are strongly convex.
+    def __init__(self, f, g, operator, tau=None, sigma=None, initial=None, gamma_g=None, gamma_fconj=None, **kwargs):
+        """Initialisation of the PDHG algorithm"""
 
+        self._theta = kwargs.pop('theta', 1.0)
+        if self.theta > 1 or self.theta < 0:
```

Comment:
Tiny issue, it's just odd seeing the parameter accessed differently on consecutive lines. I think it aids readability to either use the parameter directly i.e. `self._theta` , or add a setter so you only access `self.theta` in the code. or, pop to a local variable, test, then assign it.

Reply:
Thanks gemma - have changed it to `self._theta`